### DISCUSSION TITLE
Does this look right?

### DISCUSSION THREAD

**Discussion by MCube78** [kschan0214/sepia#72] [Q&A]
Hi, 
sorry I have a newb question. I ran SEPIA using "vanilla" settings (from the First QSM with SEPIA), configured in the SEPIA ("one-stop QSM") tab, but the output seems strange to my untrained eye. 
For reference my settings are: 
- **Phase Unwrapping:** Optimum Weights/Laplacian(MEDI)
- **BFR:** SHARP
- **QSM:** TKD (with Brain Mask for ref Tissue).

Would someone be kind enough to tell me if this looks right?

![Sepia_Chimap](https://github.com/kschan0214/sepia/assets/29030917/c490ac90-2ff6-4320-9096-8e2d71aea982)

To me there are clear big striation that I wasn't expecting. Plus, in some datasets it gets much much worse, as in this image (ignore the red outline..):

![Bad_QSM](https://github.com/kschan0214/sepia/assets/29030917/9eabe393-0cfe-4b74-b943-e5fdfdf6f45d)

Any feedback would be very much appreciated!

 Thank you,
 Martin

**Comment by kschan0214** [comment]
Hi Martin,

I should have put the information on the ducmentation website mentioning the "default" setting of SEPIA is designed for "minimal requirement of external dependency". However, these algorithms are quite basic and may not produce optimal QSM result. (the default algorithms will be updated in the future).

Most of the artifacts I observed seems to be originating from regions close to air/tissue interface, potentially caused by inaccurate skull stripping near those regions. These artefacts can be mitigated by most of the state-of-the-art methods. If you have the [mritools](https://github.com/korbinian90/CompileMRI.jl/releases/tag/v3.5.5), and [MEDI toolbox](http://pre.weill.cornell.edu/mri/pages/qsm.html) or [FANSI toolbox](https://gitlab.com/cmilovic/FANSI-toolbox/-/tree/b6ac1c9ea03380722ebe25a6dbef33fff4ea3700) available (optionally [STI Suite](https://chunleiliulab.github.io/software.html)), I would suggest using the [MEDI pipeline](https://github.com/kschan0214/QSM_Consensus_Paper_Example_Code/tree/main/SEPIA_Pipeline_MEDI) or the [FANSI pipeline](https://github.com/kschan0214/QSM_Consensus_Paper_Example_Code/tree/main/SEPIA_Pipeline_FANSI) as your starting point. (You can import one of these pipeline config files onto the SEPIA GUI.) These two pipelines follow [the recent published recommendations](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10350101/) from the EMTP study group and usually produce robust results.

Let me know if you have any questions and keep me posted!

Best wishes,

Kwok

**Reply by MCube78** [reply]
Thank you so much, will definitely try!
